src/lib/lightning/clightning/clightningService.ts

```diff
@@ -172,7 +164,7 @@ export class CLightningService implements LightningService {
   }
 
   async closeChannel(node: LightningNode, channelPoint: string): Promise<any> {
-    const body = { id: channelPoint };
+    const body = { id: channelPoint, unilateraltimeout: 1 }; // close the channel unilaterally after 1 ms
```

Comment:
Why add the `unilateraltimeout` parameter? This will trigger a force-close which was unexpected for me. The other nodes only do collaborative closes via the UI. If you want to force-close a channel, you need to use the CLI. I think it's best to keep that same behavior here.

Reply:
I added the parameter because for some weird reason, Carol goes offline immediately Bob tries to close the channel. Bob keeps trying to connect with carol, and it lingers. See logs below:

```
[electron] 17:13:49.365 › Eclair API: [stream] carol {
[electron]   type: 'channel-state-changed',
[electron]   channelId: 'be815413dffd4cd5052b00df4957bc0c9059ac693c43e01d483c591f23d830e7',
[electron]   remoteNodeId: '0340abaa44e691cff8df5f35f9e97175d603097d34b956db5060014511bd3aa2ae',
[electron]   previousState: 'OFFLINE',
[electron]   currentState: 'SYNCING'
[electron] }
[electron] 17:13:49.426 › Eclair API: [stream] carol {
[electron]   type: 'channel-state-changed',
[electron]   channelId: 'be815413dffd4cd5052b00df4957bc0c9059ac693c43e01d483c591f23d830e7',
[electron]   remoteNodeId: '0340abaa44e691cff8df5f35f9e97175d603097d34b956db5060014511bd3aa2ae',
[electron]   previousState: 'SYNCING',
[electron]   currentState: 'SHUTDOWN'
[electron] }
[electron] 17:13:49.437 › Eclair API: [stream] carol {
[electron]   type: 'channel-state-changed',
[electron]   channelId: 'be815413dffd4cd5052b00df4957bc0c9059ac693c43e01d483c591f23d830e7',
[electron]   remoteNodeId: '0340abaa44e691cff8df5f35f9e97175d603097d34b956db5060014511bd3aa2ae',
[electron]   previousState: 'SHUTDOWN',
[electron]   currentState: 'OFFLINE'
[electron] }
```
I have been investigating for a while now but still couldn't find out the issue.

Reply:
Ah so there appears to be a compatibility issue when closing CLN channels with an Eclair node. I tested with CLN->LND and CLN->CLN. Both of those were mutual closes. It makes sense to keep this parameter in there. Great work finding that.

Just one suggestion, the comment states `1 ms` but the [CLN Docs](https://docs.corelightning.org/reference/lightning-close) says `unilateraltimeout` is in seconds. So the comment just needs to be corrected.

Reply:
I have added a comment to address why we are force closing it, so it can be removed when the issue is solved.